app/budget/services/budgetExcelService.js

```diff
@@ -0,0 +1,456 @@
+class BudgetExcelService {
```

Comment:
For new files please use 2 spaces per indent. There is some inconsistency across the project as we're still in the process of migrating all the frontend files over to this format.

Also, across this file there are indentation issues and new lines that need to be added, I commented some but not all of them.